.github/workflows/tf-admin-aco-deny.yml

```diff
@@ -18,39 +18,46 @@ on:
         required: false
         type: boolean
 
+defaults:
+  run:
+    working-directory: ./terraform/services/admin-aco-deny
+
 jobs:
   check-fmt:
     runs-on: codebuild-cdap-${{github.run_id}}-${{github.run_attempt}}
     steps:
       - uses: actions/checkout@v4
-      - uses: ./actions/setup-tfenv-terraform
-      - run: terraform fmt -check -diff -recursive terraform/services/admin-aco-deny
+      - uses: sigstore/cosign-installer@d58896d6a1865668819e1d91763c7751a165e159 # v3.9.2
+      - uses: ./actions/setup-tenv # TODO Use cmsgov/cdap/actions/setup-tenv@<hash> after this lands in main
+        env:
+          TENV_GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Good attempt to fix the rate-limiting errors, but they're occuring on the tofu run steps. I think the best way to solve this will be to set the TENV_GITHUB_TOKEN env var at the top level of the workflow, above the defaults block above.